EventListener/ViewResponseListener.php

```diff
@@ -27,16 +28,22 @@
  */
 class ViewResponseListener extends TemplateListener
```

Comment:
This is causing a bug see https://github.com/FriendsOfSymfony/FOSRestBundle/issues/1301.
Should we revert this change or force the user to always enable the ``TemplateListener`` ?